moma_controllers/moma_ocs2/src/MobileManipulatorInterface.cpp

```diff
@@ -264,14 +264,14 @@ std::unique_ptr<StateCost> MobileManipulatorInterface::getSelfCollisionConstrain
   std::cerr << "SelfCollision: Testing for " << numCollisionPairs << " collision pairs\n";
 
   std::unique_ptr<StateConstraint> constraint;
-  if (usePreComputation) {
+  //if (usePreComputation) {
     constraint = std::unique_ptr<StateConstraint>(new MobileManipulatorSelfCollisionConstraint(
         MobileManipulatorPinocchioMapping<scalar_t>(armInputDim_, baseType_), std::move(geometryInterface), minimumDistance));
-  } else {
+  /*} else {
     constraint = std::unique_ptr<StateConstraint>(
         new SelfCollisionConstraintCppAd(pinocchioInterface, MobileManipulatorPinocchioMapping<scalar_t>(armInputDim_, baseType_), std::move(geometryInterface),
                                          minimumDistance, "self_collision", libraryFolder, recompileLibraries, false));
```

Comment:
A current problem is that this computation crashes the node (failing an internal assertion). Thus, I can not disable `usePreComputation`, which I would like to test.